src/uct/ib/base/ib_md.c

```diff
@@ -38,8 +38,11 @@
         ucs_status_t _status; \
         \
         UCT_MD_MEM_DEREG_CHECK_PARAMS( \
-                _params, (_ib_md)->cap_flags & UCT_MD_FLAG_INVALIDATE); \
-        _status = uct_ib_md_mem_dereg_params_invalidate_check(_params); \
+                _params, ((_ib_md)->cap_flags & (UCT_MD_FLAG_INVALIDATE | \
```

Comment:
1. why need UCT_MD_FLAG_INVALIDATE flag?
2. maybe move this check inside uct_ib_md_mem_dereg_params_invalidate_check() function, to have less code in the macro?